SimpleCalculator.py

```diff
@@ -71,32 +72,43 @@ def find_min_value(values):
 # TODO this function should take list of values, then return value with biggest value
 def find_max_value(values):
     max_value = 0
-    # TODO  here should be your implementation
+
+
     return max_value
 
 
 # TODO this function should take list of values, then return average of those values
 
+
 def calculate_average(values):
     average = 0
     average = sum(values) / len(values)
-    # TODO  here should be your implementation
     return average
 
 
 
 # TODO this function should take list of values,
 # then returns sorted list in ascending(normal) or descending(reverse" order
 def sort_values(values, order="normal"):
-     # TODO  here should be your implementation
+    for x in range(0,5):
+        for i in range(0,5):
+            if list[i] > list[i+1]:
```

Comment:
Ok, as I have written in you task description(I hope that I wrote that :D ) this function should sort both in ascending and descending order. How it should be sorted is passed in order parameter, this parameter should taken "normal" or "reverse" value, and sort them according to passed order. For now this function only sorts in ascending order, you should also add descending order, there should be tests for descending order.